continuous_integration/recipes/distributed/meta.yaml

```diff
@@ -24,28 +24,28 @@ build:
 
 requirements:
   host:
-    - python >=3.9
+    - python >=3.10
     - pip
     - dask-core {{ dask_version }}
     - versioneer =0.29
     - tomli # [py<311]
   run:
-    - python >=3.9
+    - python >=3.10
     - click >=8.0
-    - cloudpickle >=1.5.0
-    - cytoolz >=0.10.1
+    - cloudpickle >=2.0.0
+    - cytoolz >=0.11.2
     - {{ pin_compatible('dask-core', max_pin='x.x.x.x') }}
     - jinja2 >=2.10.3
     - locket >=1.0.0
-    - msgpack-python >=1.0.0
+    - msgpack-python >=1.0.2
     - packaging >=20.0
-    - psutil >=5.7.2
-    - pyyaml >=5.3.1
+    - psutil >=5.8.0
+    - pyyaml >=5.4.1
     - sortedcontainers >=2.0.5
     - tblib >=1.6.0
     - toolz >=0.10.0
-    - tornado >=6.0.4
-    - urllib3 >=1.24.3
+    - tornado >=6.2.0
```

Comment:
also doesn't match pyproject.toml

Reply:
I guess this is true for all of the above.

Reply:
Updated